### PR TITLE
Add simple navbar

### PR DESCRIPTION
**Pull request by OUMontiel**
# What does this PR do?
- Adds a simple navbar to the homepage

**Related Issues**
- #5 

---
**Merge Checklist**
- [x] Branch is up to date
- [x] No code style issues

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -20,6 +20,43 @@
...
+          </li>
+          <li class="nav-item">
+            <button type="button" class="btn btn-outline-success">Log In</button>
+          </li>
+          <li class="nav-item dropdown">
+            <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+              <svg width="30px" height="30px" viewBox="0 0 16 16" class="bi bi-inbox-fill" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" d="M3.81 4.063A1.5 1.5 0 0 1 4.98 3.5h6.04a1.5 1.5 0 0 1 1.17.563l3.7 4.625a.5.5 0 0 1-.78.624l-3.7-4.624a.5.5 0 0 0-.39-.188H4.98a.5.5 0 0 0-.39.188L.89 9.312a.5.5 0 1 1-.78-.624l3.7-4.625z"/>
+                <path fill-rule="evenodd" d="M.125 8.67A.5.5 0 0 1 .5 8.5h5A.5.5 0 0 1 6 9c0 .828.625 2 2 2s2-1.172 2-2a.5.5 0 0 1 .5-.5h5a.5.5 0 0 1 .496.562l-.39 3.124a1.5 1.5 0 0 1-1.489 1.314H1.883a1.5 1.5 0 0 1-1.489-1.314l-.39-3.124a.5.5 0 0 1 .121-.393z"/>
+              </svg>
```
Comment (abundis28) [inline src/main/webapp/index.html:51]: Is this default syntax/structure of the component of the Material.io library? Or something you wrote for a specific purpose? Either way could yo document what this is?

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -20,6 +20,43 @@
...
   </head>
   <body>
-      
+    <!-- Navbar -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <!-- Logo -->
+      <a class="navbar-brand" href="#">
+        <img src="images/mintern-logo.png" height="50" alt="Mintern logo">
+      </a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggler" aria-controls="navbarToggler" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (abundis28) [inline src/main/webapp/index.html:29]: Consider wrapping this line to improve readability.

File: src/main/webapp/index.html (hunk 1 above)
Comment (OUMontiel) [reply src/main/webapp/index.html:51]: Sure, I'll document it! This was before I made the Material.io suggestion, so this is still Boostrap. It's just a simple icon, and that's how they tell you to write it, I only modified the width and height attributes.

File: src/main/webapp/index.html (hunk 2 above)
Comment (OUMontiel) [reply src/main/webapp/index.html:29]: Done!

**Pull request merged**